### DISCUSSION TITLE
Some type of ⌘ + number shortcut support

### DISCUSSION THREAD

**Discussion by kylecotter** [trendyminds/craft-palette#44] [Ideas]
Similar to Alfred App, would be great to map results to command key shortcuts so one could ⌘ + 1, or ⌘ + 3 etc for results that have multiple items so you don't need to arrow up or down.

I know ⌘ + num is already mapped to browser tab switching, but some similar type of shortcut here would be great!

**Comment by aaronbushnell** [comment]
Cool idea, @kylecotter! I may take a stab at this at some point
